br-webui/views/system.liquid

```diff
@@ -337,5 +349,38 @@ socket.on('cpu stats', function(data) {
 		</div>
 	</div>
 </div>
+<div class="row">
+        <div class="col-md-12">
+                <div class="panel panel-default">
+                        <div class="panel-heading">
+                                <h3 class="panel-title">
+                                        Status of Companion Processes
+                                </h3>
+                        </div>
+                        <div class="panel-body">
+                                <dl class="dl-horizontal">
+                                        <dt>mavproxy: </dt>
+                                                <dd id="mav"></dd>
+                                        <dt>video: </dt>
+                                                <dd id="vid"></dd>
+                                        <dt>webterminal: </dt>
+                                                <dd id="webterm"></dd>
+                                        <dt>audio: </dt>
+                                                <dd id="aud"></dd>
+                                        <dt>webui: </dt>
+                                                <dd id="web"></dd>
+					<dt>file-manager: </dt>
+                                                <dd id="filemanager"></dd>
+					<dt>commrouter: </dt>
+                                                <dd id="router"></dd>
+					<dt>nmearx: </dt>
+                                                <dd id="nmearx"></dd>
+					<dt>wldriver: </dt>
+                                                <dd id="driver"></dd>
```

Comment:
Put the list in alphabetical order and organize the indentation of the itens.
```html
                                        <dt>audio: </dt>
                                                <dd id="aud"></dd>
                                        <dt>webui: </dt>
                                                <dd id="web"></dd>
					<dt>file-manager: </dt>
                                                <dd id="filemanager"></dd>
					<dt>commrouter: </dt>
```

Reply:
What do you mean by organize the Indentation?